config/sync/registration_validation.settings.yml

```diff
@@ -0,0 +1,7 @@
+domains: ''
+usernames: "admin\r\nroot\r\nwebmaster\r\nwebadmin\r\npostmaster\r\nadministration\r\nadmin"
```

Comment:
Is there a reason admin is added in twice and also should administrator not be added in this list?

Reply:
You mean it shouldn't be added in this list?

Reply:
No i mean it's added twice in the list. The first and last value both say "admin".

Reply:
omg Lincoln I mean administration

Reply:
It should be added in the list.

Reply:
well it is. So I'm pushing now.

Reply:
sorry I misread your earlier comment. It is done.